dogowner/models.py

```diff
@@ -0,0 +1,78 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Gender(models.TextChoices):
+    Male = 'M', 'Male'
+    Female = 'F', 'Female'
+    Unknown = 'UN', 'Unknown'
+
+
+class Dogowner(models.Model):
+
+    user = models.ForeignKey(User,
+                             on_delete=models.CASCADE,
+                             blank=True,
+                             null=True
+                             )
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    phone_number = models.CharField(max_length=20)
+    dog_name = models.CharField(max_length=50)
+    dog_race = models.CharField(max_length=50)
+    dog_picture_url = models.CharField(max_length=255)
+    dog_age = models.IntegerField()
+    dog_weight = models.FloatField()
+    dog_gender = models.CharField(
+        max_length=2,
+        choices=Gender.choices,
+        default='M',)
+
+    DogOwners = models.Manager()
+
+    def __str__(self):
+        return self.first_name + self.last_name
+
+    @staticmethod
+    def create_dogowner(email, username, password, dog_name,
```

Comment:
Why do you need this method?

Reply:
I had to find a way for init Django user with our requierd fields , and make sure we will not have a situation of creating dog- owner without user initalizion.
That is why I assumed this method can help .

Reply:
I will try to search for "cleaner" way for this task .

Reply:
According to https://github.com/redhat-beyond/Hotails/pull/33#discussion_r833328410 , I gave up on this method and we will use regular initializion.
I guess because we don't have any registration in our application ,we can make sure that "User" entity will initalize with our required fields .